x-pack/plugin/searchable-snapshots/build.gradle

```diff
@@ -27,3 +27,17 @@ gradle.projectsEvaluated {
     .findAll { it.path.startsWith(project.path + ":qa") }
     .each { check.dependsOn it.check }
 }
+
+configurations {
+  testArtifacts.extendsFrom testRuntime
+}
+
+task testJar(type: Jar) {
+  appendix 'test'
+  from sourceSets.test.output
+  include '**/*TestCase.class'
```

Comment:
Is this filter strictly required? This opens up potential issues with test cases referencing other classes in the test source set that don't match this pattern and causing things to explode at runtime.

Reply:
Good catch again, thanks